### ISSUE TITLE
Fix Description of helper Functions (Change argv to args)

### ISSUE THREAD

**Issue by HenrikWittemeier** [aquasecurity/tracee#4302]
https://github.com/aquasecurity/tracee/blob/d09d7fca0d53a19052435536a2290ed8415472d1/signatures/helpers/arguments_helpers.go#L47

https://github.com/aquasecurity/tracee/blob/d09d7fca0d53a19052435536a2290ed8415472d1/signatures/helpers/arguments_helpers.go#L61

https://github.com/aquasecurity/tracee/blob/d09d7fca0d53a19052435536a2290ed8415472d1/signatures/helpers/arguments_helpers.go#L75

https://github.com/aquasecurity/tracee/blob/d09d7fca0d53a19052435536a2290ed8415472d1/signatures/helpers/arguments_helpers.go#L89

given from the event "argv" field,

In my understanding the comment should be: [...] given from the event "**args**" field, [...].
Its just a small Issue but it helps understanding things a lot :)

**Comment by geyslan** [comment]
@HenrikWittemeier tks for reporting this. @ShohamBit perhaps this is related to your current effort. LMK.

**Comment by ShohamBit** [comment]
Yes @HenrikWittemeier  you are very right it is a bit confusing, so your suggestions is to replcae "argv" to "args" rigth?

**Comment by HenrikWittemeier** [comment]
Yes right! @ShohamBit

**Comment by ShohamBit** [comment]
hey @HenrikWittemeier i created a pull request that fix 
See pull request #4304  for the fix.
